src/openeo_pg_parser/processes/absolute.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
is this a copy of the openeo processes repository? 
Isn't it better to use git submodules for this?

in the openeo-python-driver we are using git submodules: here https://github.com/Open-EO/openeo-python-driver/tree/master/openeo_driver/specs/openeo-processes

Reply:
We were discussing recently to publish them as package as submodules sometimes are really annoying to use. Would that be useful in general?

Reply:
> publish them as package

You mean as a zip/tar file?
While I understand that git submodules take some to time to get used to,
I'm not sure whether separate packages simplify the setup/build procedure. With git submodules you just have to do something like `git submodule update` and you're set. While with a generic package you have to manage some public URL with versions/tags, download from that URL, make sure you have correct tools to upack it, unpack to correct location, etc. Not impossible, but a lot more tedious and error prone to do manually or to automate. Git submodules eliminate a lot of those troubles (at the expense of being harder to understand)

Reply:
No, I meant as NPM / PyPi packages so that they can used with the normal dependency management etc...

Reply:
I would agree with @m-mohr. If we aim to publish a version for each package manager (npm and pip at the moment), isn't it cleaner to do this without git submodules?

Reply:
ok, npm/PyPI packages would indeed be a handy replacement for git submodules indeed.

One disadvantage I can think of is that you then can only "include" one specific version of the openeo-processes. In the openeo-python-driver for example we currently both support 0.4 and 1.0 versions, which is possible with git submodules. But most other applications will only need to link to a single version I guess.

Reply:
That is indeed true and not sure whether there's an easy way to provide all versions through a package...

Reply:
Thanks both for your input. So whats the conclusion on this, should we stick to a semi-dynamic version of the processes (always updated with new releases of the parser) or use a git submodule? For the the time being I would prefer the first option, because this would be also easier to handle from the testing side at the moment. As soon as I have more time again, I can have a look into this and replace the process folder with a git submodule.